### ISSUE TITLE
System.TypeLoadException: Could not load type 'Deedle.FSharpFrameExtensions' from assembly 'Deedle, Version=1.2.4.0, Culture=neutral, PublicKeyToken=null'.

### ISSUE THREAD

**Issue by rhscjohn** [TahaHachana/XPlot#31]
The complete error is:

System.TypeLoadException: Could not load type 'Deedle.FSharpFrameExtensions' from assembly 'Deedle, Version=1.2.4.0, Culture=neutral, PublicKeyToken=null'.
   at XPlot.GoogleCharts.Deedle.Chart.Table.Static[K,V](Frame`2 data, FSharpOption`1 Options)
   at <StartupCode$FSI_0040>.$FSI_0040.main@() in C:\Users\John\documents\visual studio 2015\Projects\DeedleTutorial\DeedleTutorial\Tutorial.fsx:line 190
Stopped due to error

The source code:
`// Chart type: Table of Frame - cl
cl
    |> Chart.Table
    |> Chart.WithOptions(Options(showRowNumber = true, page = "enable",    pageSize = 20)) `

IDE: Visual Studio 2015 Community Edition.
Installed FsLab via Paket.
Paket dependencies file contents:
source https://nuget.org/api/v2

nuget FsLab

paket.lock file contents:
NUGET
  remote: https://www.nuget.org/api/v2
  specs:
    Deedle (1.2.4)
    Deedle.RPlugin (1.2.4)
      Deedle (>= 1.2.4)
      R.NET.Community (>= 1.6.4)
      R.NET.Community.FSharp (>= 1.6.4)
      RProvider (>= 1.1.13)
    DynamicInterop (0.7.4)
    Foogle.Charts (0.0.5)
    FSharp.Charting (0.90.13)
    FSharp.Core (4.0.0.1)
    FSharp.Data (2.2.5)
      Zlib.Portable (>= 1.10.0) - framework: portable-net40+sl50+wp80+win80
    FsLab (0.3.17)
      Deedle (1.2.4)
      Deedle.RPlugin (1.2.4)
      DynamicInterop (0.7.4)
      Foogle.Charts (0.0.5)
      FSharp.Charting (0.90.13)
      FSharp.Data (2.2.5)
      Google.DataTable.Net.Wrapper (3.1.2)
      MathNet.Numerics (3.10.0)
      MathNet.Numerics.FSharp (3.10.0)
      Newtonsoft.Json (8.0.2)
      R.NET.Community (1.6.4)
      R.NET.Community.FSharp (1.6.4)
      RProvider (1.1.15)
      XPlot.GoogleCharts (1.3.1)
      XPlot.GoogleCharts.Deedle (1.3.1)
      XPlot.Plotly (1.3.1)
    Google.DataTable.Net.Wrapper (3.1.2)
    Http.fs (1.5.1)
    MathNet.Numerics (3.10.0)
      TaskParallelLibrary (>= 1.0.2856) - framework: net35
    MathNet.Numerics.FSharp (3.10.0)
      FSharp.Core (>= 3.1.2.5)
      MathNet.Numerics (3.10.0)
    Newtonsoft.Json (8.0.2)
    R.NET.Community (1.6.4)
      DynamicInterop (0.7.4)
    R.NET.Community.FSharp (1.6.4)
      R.NET.Community (>= 1.6.4)
    RProvider (1.1.15)
      DynamicInterop (0.7.4)
      R.NET.Community (1.6.4)
      R.NET.Community.FSharp (1.6.4)
    TaskParallelLibrary (1.0.2856) - framework: net35
    XPlot.GoogleCharts (1.3.1)
      Google.DataTable.Net.Wrapper
      Newtonsoft.Json
    XPlot.GoogleCharts.Deedle (1.3.1)
      Deedle
      XPlot.GoogleCharts (>= 1.3.1)
    XPlot.Plotly (1.3.1)
      Http.fs
      Newtonsoft.Json
    Zlib.Portable (1.11.0) - framework: portable-net40+sl50+wp80+win80

I found found this on [https://gist.github.com/larjo/252ab41979459fa80a50](url)
I also found this this on StackOverflow [http://stackoverflow.com/questions/34180107/xplot-and-deedle-in-f)

**Comment by TahaHachana** [comment]
Did you load the FsLab.fsx script before executing your code and can you share the problematic lines please?

**Comment by rhscjohn** [comment]
Hi,
Yes, I did load the FsLab.fsx script.  Attached is a screen shot of the code.
When I plot a Deedle Series there is no problem.  See the output in the web browser.  That was produced by running:
// Chart type: Table of Series (column -Close)closes    |> Chart.Table    |> Chart.WithOptions(Options(showRowNumber = true, page = "enable",    pageSize = 20))     |> Chart.WithLabels ["Date"; "Close"]
The variable closes was generated with:  let closes =  cl?Close

When I plot a Deedle Frame I get the System.TypeLoadException error.  The code for plotting the frame:
// Chart type: Table of Frame - clcl    |> Chart.Table    |> Chart.WithOptions(Options(showRowNumber = true, page = "enable",    pageSize = 20)) 
The variable cl was generated with: let cl = Deedle.Frame.ReadCsv<DateTime>(root + "../Data/CL 11-15 60 Min Dec 28 #5.PRN", separators="\t", indexCol="Date")    I have read that at some point -  'Deedle.FSharpFrameExtensions' got changed to-  'Deedle.FSharp Frame Extensions'.  But I have not been able to confirm that.  This change may have occurred in Deedle version 1.2.4.
John
John

Date: Sun, 3 Apr 2016 06:59:12 -0700
